File: conference/registration-bc/web/src/main/java/com/microsoft/conference/registration/domain/SeatAssigning/Events/SeatUnassigned.cs
using System;
using ENode.Eventing;

namespace Registration.SeatAssigning
{
    public class SeatUnassigned : DomainEvent<Guid>
    {
        public int Position { get; private set; }

        public SeatUnassigned() { }
        public SeatUnassigned(int position)
        {
            Position = position;
        }
    }
}

File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/domain/seatassigning/Events/SeatUnassigned.java
package com.microsoft.conference.registration.domain.seatassigning.Events;

import org.enodeframework.eventing.DomainEvent;

public class SeatUnassigned extends DomainEvent<String> {
    public int position;

    public SeatUnassigned() {
    }

    public SeatUnassigned(int position) {
        this.position = position;
    }
}
